src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
fix indentation according to the code style

Reply:
fix it in all places